pkg/metadata/inventories/README.md

```diff
@@ -36,19 +36,74 @@ The payload is a JSON dict with the following fields
+  - `agent_version` - **string**: the version of the Agent.
+  - `flavor` - **string**: the flavor of the Agent. The Agent can be build under different flavor such as standalone
+    dogstatsd, iot, serverless ... (see `pkg/util/flavor` package).
+  - `config_apm_dd_url` - **string**: the configuration value `apm_config.dd_url` (scrubbed)
+  - `config_dd_url` - **string**: the configuration value `dd_url` (scrubbed)
+  - `config_site` - **string**: the configuration value `site` (scrubbed)
+  - `config_logs_dd_url` - **string**: the configuration value `logs_config.logs_dd_url` (scrubbed)
+  - `config_logs_socks5_proxy_address` - **string**: the configuration value `logs_config.socks5_proxy_address` (scrubbed)
+  - `config_no_proxy` - **array of strings**: the configuration value `proxy.no_proxy` (scrubbed)
+  - `config_process_dd_url` - **string**: the configuration value `process_config.process_dd_url` (scrubbed)
+  - `config_proxy_http` - **string**: the configuration value `proxy.http` (scrubbed)
+  - `config_proxy_https` - **string**: the configuration value `proxy.https` (scrubbed)
+  - `install_method_tool` - **string**: the name of the tool used to install the agent (ie, Chef, Ansible, ...).
+  - `install_method_tool_version` - **string**: the tool version used to install the agent (ie: Chef version, Ansible
+    version, ...). This defaults to `"undefined"` when not installed through a tool (like when installed with apt, source
+    build, ...).
+  - `install_method_installer_version` - **string**:  The version of Datadog module (ex: the Chef Datadog package, the Datadog Ansible playbook, ...).
+  - `logs_transport` - **string**:  The transport used to send logs to Datadog. Value is either `"HTTP"` or `"TCP"` when logs collection is
+    enabled, otherwise the field is omitted.
+  - `feature_cws_enabled` - **bool**: True if the Cloud Workload Security is enabled (see: `runtime_security_config.enabled`
+    config option).
+  - `feature_process_enabled` - **bool**: True if the Process Agent is enabled (see: `process_config.enabled` config
+    option).
+  - `feature_networks_enabled` - **bool**: True if the Network Performance Monitoring is enabled (see:
+    `network_config.enabled` config option in `system-probe.yaml`).
+  - `feature_logs_enabled` - **bool**: True if the logs collection is enabled (see: `logs_enabled` config option).
+  - `feature_cspm_enabled` - **bool**: True if the Cloud Security Posture Management is enabled (see:
+    `compliance_config.enabled` config option).
+  - `feature_apm_enabled` - **bool**: True if the APM Agent is enabled (see: `apm_config.enabled` config option).
+
+("scrubbed" indicates that secrets are removed from the field value just as they are in logs)
 
 ## Example Payload
 
 Here an example of an inventory payload:
 
 ```
 {
-    "agent_metadata": {
-        "hostname_source": "os"
-    },
+   "agent_metadata": {
```

Comment:
The `cloud_provider` value is missing here.  It is optional, but might be best to have it appear anyway.